pkg/kdump/config-client.es6

```diff
@@ -0,0 +1,165 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+
+/* Parse an ini-style config file
+ * and monitor it for changes
+ */
+export class ConfigFile {
+    constructor(filename, superuser = false) {
+        this.filename = filename;
+        this._rawContent = undefined;
```

Comment:
There is currently no concept of a `private` variable. Even with [workarounds](http://www.2ality.com/2016/01/private-data-classes.html) the data is still technically accessible.